x-pack/legacy/plugins/maps/public/selectors/map_selectors.js

```diff
@@ -34,7 +36,7 @@ function createLayerInstance(layerDescriptor, inspectorAdapters) {
 
   switch (layerDescriptor.type) {
     case TileLayer.type:
-      return new TileLayer({ layerDescriptor, source });
+      return new TileLayer({ layerDescriptor, source, style: new TileStyle() });
```

Comment:
Should we just instantiate a new style in the `TileLayer` constructor if none was passed in?